### PR TITLE
gunicorn and threads options

### PR DESCRIPTION
**Pull request by Guillaume-sousa**
- Added option to switch between gunicorn and Uwsgi,
- Added Nginx files for gunicorn upstream,
- Added options to set number of processes and threads via environment.

### CODE REVIEW

**Review by pdessauw — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -83,7 +83,9 @@ and filled.
 | REDIS_VERSION         | Version of the Redis image |
 | POSTGRES_VERSION      | Version of the Postgres image |
 | NGINX_VERSION         | Version of the NGINX image |
-| UWSGI_PROCESSES       | Number of uwsgi processes to start (default 10) |
+| WEB_SERVER            | Web server for the CDCS (e.g. `uwsgi`, `gunicorn`)
+| PROCESSES             | Number of uwsgi processes (default `--processes=8`) / gunicorn workers to start (default `workers=cpu_count() * 2 + 1`) |
+| THREADS               | Number of uwsgi/gunicorn threads (default 8)|
```
Comment (MyleneSimon) [inline README.md]: The description of `THREADS` could be improved by explaining that number of threads is per process/worker

File: README.md [hunk 2]
```diff
@@ -258,6 +260,21 @@ default the `docker-compose` file sets it using the values of
 For more information about production deployment of a Django project,
 please check the [Deployment Checklist](https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/#deployment-checklist)
 
+#### Web Server
+
+The following web servers are available for the CDCS: Uwsgi and Gunicorn. 
+
+The CDCS image contains a default configurations for each:
+- The default Uwsgi configuration writes in a UNIX socket, that Nginx reads from.
```
Comment (MyleneSimon) [inline README.md]: `Uwsgi` in the documentation should be replaced by `uWSGI`

**Review by MyleneSimon — COMMENTED** [review]
Looks good, I only have minor change suggestions for the documentation

**Review by MyleneSimon — APPROVED** [review]

**Pull request merged**